public/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, user-scalable=no" />
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <title aria-hidden="true">VotingWorks Ballot Marking Device</title>
+    <title aria-hidden>VotingWorks Ballot Marking Device</title>
```

Comment:
@benadida - I'm fairly certain that this has no affect. When first loading the web app, the page title is always read aloud.

Reply:
removing.